index.html

```diff
@@ -559,6 +563,7 @@ <h1>{{ chosen_domain == 'US' ? 'USA' :
      select, updateSelect) {
   scale = scale || 'log10';
   if (chart) { chart.detach(); }
+  var isGrowthChart = (stat == 'growth');
```

Comment:
I tried to reduce the number of places we test for the literal string, but still ended up with a few. Ideally, we'd have a more formal data model to avoid dealing with string settings all the way through.

Reply:
OK.  I think this hack is clean enough for now.